test/compilable/b21285.d

```diff
@@ -0,0 +1,20 @@
+// Issue 21285 - Delegate covariance broken between 2.092 and 2.094 (git master). 
```

Comment:
Should have?
`// REQUIRED_ARGS: -unittest`

Reply:
You're probably right. Sorry about this taking a while.